app/Lib/Rscripts/analyse.R

```diff
@@ -2,11 +2,45 @@
 library(psy)
 
 Analyse <- function(key, input.answers, number.answeroptions) {
+  # Calculates standard psychometric properties of an exam, notably the 
+  # percentage correct and the item rest correlation (IRC) per item and
+  # per answer option and the cronbach's alpha for the whole exam.
+  #
+  # Arguments:
```

Comment:
"(...) a list of the function's arguments, denoted by Args: (...)" see: https://google.github.io/styleguide/Rguide.xml#functiondocumentation